### ISSUE TITLE
AARCH64 Support?

### ISSUE THREAD

**Issue by OliverEvans96** [PyYoshi/cChardet#22]
Hello!

I'm using an ODROID C2 (http://www.hardkernel.com/main/products/prdt_info.php), which has AARCH64 architecture. The operating system is Ubuntu MATE 16.04.

When running `pip install cchardet`, I get this error complaining about the architecture:

[cchardet_error.txt](https://github.com/PyYoshi/cChardet/files/697684/cchardet_error.txt)

The same happens when I try to install manually from this repository. I found the following, which seems like what I need: http://tardis.tiny-vps.com/aarm/packages/p/python-cchardet/ but I'm not sure how to install it, as it doesn't contain a setup.py file.

Any help is appreciated!

Thanks,
Oliver

**Comment by PyYoshi** [comment]
Thank you for reporting.

I've never been to try it on aarch64 yet.
But I found aarch64 support patch for cchardet.

Can you try [this patch](https://github.com/archlinuxarm/PKGBUILDs/blob/master/community/python-cchardet/0001-Add-AArch64-support.patch)?

Thanks.

**Comment by OliverEvans96** [comment]
PyYoshi,

Thanks so much for the response! With your patch, the install works fine, but I now get the following error upon importing cchardet:

```
odroid@odroid64:~/local/src/cChardet (master*%=) % python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import cchardet
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/cchardet/__init__.py", line 1, in <module>
    from cchardet import _cchardet
ImportError: dynamic module does not define init function (init_cchardet)
```

I'm trying to set up a [zulip](https://github.com/zulip/zulip) server on the odroid, and zulip uses cchardet. If it's not possible to get cchardet working, then I'll look into modifying zulip to use chardet.

Thanks again!
Oliver

**Comment by OliverEvans96** [comment]
PyYoshi,

I was able to set up zulip with pure-python chardet.

Thank you for your time!
Oliver

**Comment by PyYoshi** [comment]
I ordered an armv8 board last week. I will try checking this issue when it arrives. :)

**Comment by OliverEvans96** [comment]
Awesome, thank you!

Oliver

**Comment by PyYoshi** [comment]
Hi @OliverEvans96 

I released [v1.1.3](https://pypi.python.org/pypi/cchardet/1.1.3).

It supports aarch64. Please check it.

Thanks.

**Comment by OliverEvans96** [comment]
Thanks so much!
